src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7478,4 +7478,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_StructLayoutCyclePrimaryConstructorParameter" xml:space="preserve">
     <value>Struct primary constructor parameter '{0}' of type '{1}' causes a cycle in the struct layout</value>
   </data>
+  <data name="ERR_UnexpectedParameterList" xml:space="preserve">
+    <value>Unexpected parameter list.</value>
```

Comment:
Error message is fine.  But consider "Interfaces cannot have primary constructors" as an alternative.

Reply:
> Error message is fine.  But consider "Interfaces cannot have primary constructors" as an alternative.

I think the message is clear enough and without additional details there is a chance that we can reuse it elsewhere under different conditions